### DISCUSSION TITLE
有群吗？想加入进去，我也是搞java的，想像博主学习

### DISCUSSION THREAD

**Discussion by darvenking** [linweiyuan/go-chatgpt-api#187] [General]
有群吗？想加入进去，我也是搞java的，想像博主学习

**Comment by linweiyuan** [comment]
然而并没有

**Reply by darvenking** [reply]
好吧，提个建议：
这儿可以这样改，就能匹配所有请求方式了：
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/10860818/3dbaf2c8-9ce4-414d-a06c-fd28bac518ba)

另外作者可以建个tg群吗？

**Reply by linweiyuan** [reply]
不建群，不进群

**Reply by linweiyuan** [reply]
文档里有我微信，有需要可以加
